src/shared/experience/expSite.ts

```diff
@@ -7,6 +7,7 @@
 import fs from 'node:fs';
 import path from 'node:path';
 import { Org, SfError } from '@salesforce/core';
+import axios from 'axios';
```

Comment:
Is there a reason we used axios over native node fetch (just curious).

Reply:
not really, it was just a little easier to get working initially. Axios has some nice extra features that help with debugging like interceptors. But yeah I could switch over to using fetch I guess